src/Wallabag/CoreBundle/Helper/Redirect.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Wallabag\CoreBundle\Helper;
+
+use Symfony\Component\Routing\Router;
+
+class Redirect
+{
+    private $router;
+
+    public function __construct(Router $router)
+    {
+        $this->router = $router;
+    }
+
+    /**
+     * @param string $url      URL to redirect
+     * @param string $fallback Fallback URL if $url is null
+     *
+     * @return string
+     */
+    public function to($url, $fallback = '')
+    {
+        $returnUrl = $url;
+
+        if (null === $url) {
+            if ('' !== $fallback) {
+                $returnUrl = $fallback;
+            } else {
+                $returnUrl = $this->router->generate('homepage');
```

Comment:
I would rather do:

```php
if (null !== $url) {
    return $url;
}

if ('' === $fallback) {
    return $this->router->generate('homepage');
}

return $fallback;
```

Also, don't forget to add a description of this helper class (before the class definition)